docs/getting-started/endpoint-diagnostic-data.asciidoc

```diff
@@ -0,0 +1,15 @@
+[[endpoint-diagnostic-data]]
+= Turn off {endpoint-sec} diagnostic data
+
+By default, {endpoint-sec} streams diagnostic data to your cluster, which Elastic uses to tune protection features. You can stop producing this diagnostic data by configuring the advanced settings in the {endpoint-sec} integration policy.
+
+NOTE: Kibana also collects usage telemetry, which includes {endpoint-sec} diagnostic data. You can control Kibana telemetry in {kibana-ref}/telemetry-settings-kbn.html[Advanced Settings].
```

Comment:
Are we officially calling the telemetry feature "Kibana telemetry"? I thought it was just "telemetry".

Reply:
@nastasha-solomon This wasn't intended as an official name of the feature, just a short description. Maybe "telemetry in Kibana"?
```suggestion
NOTE: Kibana also collects usage telemetry, which includes {endpoint-sec} diagnostic data. You can control telemetry in Kibana in {kibana-ref}/telemetry-settings-kbn.html[Advanced Settings].
```

Reply:
I _think_ it's just `telemetry` too.

Reply:
Revised the suggestion for just "telemetry"